LiskHQ/lisk-sdk
# Update API client to return JSON format - Closes #7189

### What was the problem?

This PR resolves #7189

### How was it solved?

- Update APIClient to return JSON
- Fix commander where it's using APIClient
- Fix plugin where it's using APIClient

### How was it tested?
- Check if plugin is working on the example app
- Check transaction and block related commands are working on example app
